posts/2021-08-30-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc

```diff
@@ -0,0 +1,212 @@
+
+== OpenShift Do (odo)
+
+link:https://odo.dev[Odo] is a simple CLI tool that you use to create devfile based components that interact directly with your Kubernetes cluster to setup the environment, build/deploy/access/debug your application, and link services to odo components. Directives to manage the environment and application are provided by a component's devfile.
+
+== Service Binding Operator
+
+The link:https://github.com/redhat-developer/service-binding-operator/blob/master/README.md[Service Binding Operator] makes it easier for developers to bind operator managed services (i.e. databases) to applications. More specifically, the operator will collect service data, deploy resources (i.e. secrets, configmaps) containing that data, and will make the data available to the application through the ServiceBinding custom resource it provides.
+
+== Try it out
+
+To begin, you need three things:
+
+- Kubernetes cluster. For this blog, we will be using an Openshift cluster. Be sure to log in. Odo will interact with your kubernetes cluster.
+- Openshift Do. If you have not already installed odo, do so now by following the instructions outlined in the link:https://odo.dev[odo] documentation. Be sure to install version 2.2.4 and above.
+- An application that uses a PostgreSQL database. For this blog, we will be using the Open Liberty stack link:https://github.com/OpenLiberty/application-stack-samples/tree/main/jpa[sample JPA application]
+
+
+Install the needed operators.
+
+If you have the right priviledges, you can use the OpenShift console to install these operators. You do this by navigating to `Operators->OperatorHub` and searching/selecting/installing a couple of operators from the catalog.
+
+- Service Binding Operator (provided by Red Hat)
+
+Use the OpenShift console (`Operators->OperatorHub`) to install this operator. Be sure to install version 0.9.1 and above.
+
+- PostgreSQL Operator (provided by Dev4Ddevs.com)
+
+Before installing this operator, create the project/namespace under which the application will be deployed. This operator is namespace scoped, so it needs to be installed in the same namespace where the application will be deployed.
+
+[source,sh]
+----
+$ odo project create service-binding-demo
+----
+
+Use the OpenShift console (`Operators->OperatorHub`) to install this operator. Be sure to pick `service-binding-demo` as the namespace in which to install it.
+
+Clone the application repository.
+
+[source,sh]
+----
+$ git clone https://github.com/OpenLiberty/application-stack-samples.git && \
+  cd application-stack-samples/jpa
+----
+
+Create a Java Open Liberty component.
+
+If you want the application to be built and deployed using Maven:
+
+[source,sh]
+----
+$ odo create java-openliberty mysboproj
+----
+
+If you want the application to be built and deployed using Gradle:
```

Comment:
This could look like a separate step you have to follow unless you read carefully.    

Plus it might get someone thinking.. "do I want Maven or Gradle?" but there's no reason here to care. 

Maybe just show the Maven instructions and mention at the end you could have also used Gradle?